app/components/floating_text_input_label/__snapshots__/floating_text_input.test.tsx.snap

```diff
@@ -0,0 +1,130 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FloatingTextInput test snapshot 1`] = `
+<View
+  accessible={true}
+  focusable={false}
+  onClick={[Function]}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+  style={
+    Array [
+      Object {
+        "width": "100%",
+      },
+      undefined,
+    ]
+  }
+  testID="text_input.container"
+>
+  <Text
+    animatedStyle={
+      Object {
+        "value": Object {
+          "backgroundColor": "#ffffff",
```

Comment:
seems like a lot of hard-coded colors here. @enahum shouldn't these be theme colors as well?

Reply:
@matthewbirtch you are looking at a snapshot test 🤣

Reply:
that's hilarious....sorry - I didn't even realize. Ignore me!